packages/client/bin/utils.ts

```diff
@@ -125,7 +125,7 @@ export function getArgs(): ClientOpts {
       })
       .option('mergeForkIdPostMerge', {
         describe:
-          'Place mergeForkIdTransition hardfork before (false) or after (true) Merge hardfork in the custom gethGenesis',
+          'Place mergeNetsplitBlock hardfork before (false) or after (true) Merge hardfork in the custom gethGenesis',
```

Comment:
Nit: should we rename this client option? (And what was this supposed to do? :sweat_smile: )

Reply:
We should definitely remove. I missed that.